test/driver/standalone_run_as_ppx/dune

```diff
@@ -0,0 +1,13 @@
+(library
```

Comment:
We can probaly merge the library into the executable here, what do you think? Not that important but it makes the test setup a little bit simpler, fewer files to browse, etc...

Reply:
That's definitely better. Thanks!